marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -201,37 +248,36 @@ instance ToRow UtxoRow where
     , toField (u ^. urUtxo . inlineScriptHash)
     , toField (u ^. urSlotNo)
     , toField (u ^. urBlockHash)
-    ]
+    )
 
 instance FromRow UtxoRow where
   fromRow = UtxoRow
-      <$> (Utxo <$> field <*> field <*> field <*> field
-                <*> field <*> field <*> field <*> field)
+      <$> (Utxo <$> field
+           <*> field <*> field
+           <*> field <*> field <*> field <*> field <*> field)
       <*> field <*> field
 
 instance FromRow Spent where
   fromRow = Spent <$> field <*> field <*> field <*> field
 
 instance ToRow Spent where
   toRow s =
-    [ toField (s ^. sTxInTxId)
-    , toField (s ^. sTxInTxIx)
+    [ toField (s ^. sTxId)
+    , toField (s ^. sTxIx)
     , toField (s ^. sSlotNo)
     , toField (s ^. sBlockHash)
     ]
 
----------------------------------------------------------------------------------
-------------------------------- End sql mappings ---------------------------------
-
 -- | Open a connection to DB, and create resources
 -- The parameter ((k + 1) * 2) specifies the amount of events that are buffered.
 -- The larger the number, the more RAM the indexer uses. However, we get improved SQL
 -- queries due to batching more events together.
 open
-  :: FilePath -- ^ sqlite file path
-  -> Depth    -- ^ The Depth parameter k, the larger K, the more RAM the indexer uses
+  :: FilePath   -- ^ sqlite file path
+  -> Depth      -- ^ The Depth parameter k, the larger K, the more RAM the indexer uses
+  -> Bool       -- ^ weather to perform vacuum
```

Comment:
if this change is accepted, I'd like to added to the CLI parameters for both:
- marconi-chain-index
- marconi-sidechain
on a separate PR

Reply:
You can create a story for that.

Reply:
Also, we don't want that for `marconi-sidechain` as they need to know all the UTXOs that ever occurred, even the ones that are spent on the latest slot.

Reply:
the switch is set at utxoworker in indexers.hs and covers this case.
This will change once we add the CLI parameter such that users will control that